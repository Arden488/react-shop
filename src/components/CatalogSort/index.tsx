import * as React from 'react';

const CatalogSort = () => (
  <div>
    <div>
      <div>Sort by:</div>
      <div>
        <select name="" placeholder="Select sorting">
          <option value="">No sorting</option>
          <option value="price_asc">Price ascending</option>
          <option value="name_asc">Name ascending</option>
          <option value="price_desc">Price descending</option>
          <option value="name_desc">Name Descending</option>
        </select>
      </div>
    </div>
  </div>
);

export default CatalogSort;
