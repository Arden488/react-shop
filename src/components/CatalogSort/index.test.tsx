import { shallow } from 'enzyme';
import * as React from 'react';

import CatalogSort from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<CatalogSort />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
