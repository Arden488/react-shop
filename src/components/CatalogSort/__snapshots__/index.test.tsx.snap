// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div>
      Sort by:
    </div>
    <div>
      <select
        name=""
        placeholder="Select sorting"
      >
        <option
          value=""
        >
          No sorting
        </option>
        <option
          value="price_asc"
        >
          Price ascending
        </option>
        <option
          value="name_asc"
        >
          Name ascending
        </option>
        <option
          value="price_desc"
        >
          Price descending
        </option>
        <option
          value="name_desc"
        >
          Name Descending
        </option>
      </select>
    </div>
  </div>
</div>
`;
